composer.json

```diff
@@ -31,13 +33,12 @@
         "symfony/translation": "^2.7"
     },
     "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.2",
```

Comment:
Why not use recent version of php-cs-fixer ? (last is 3.14)

Reply:
That's gonna be in the next steps, once we will have a working Docker stack to enable us to do this kind of upgrades. :wink: